### ISSUE TITLE
Breadcrumbs: clicking on a path segment for a file on git index opens empty picker

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#115397]
Steps to Reproduce:

1. stage file to index
2. click on path segment in breadcrumb

![image](https://user-images.githubusercontent.com/900690/106276971-a45a3d00-6238-11eb-8df4-76393c33fce8.png)

**Comment by jrieken** [comment]
The problem is there is nothing to show and that I don't know that... It can only show a placeholder message

**Comment by bpasero** [comment]
Yeah i think a placeholder is fine, given the thing is clickable. Or if possible not show it clickable at all.

**Comment by susiwen8** [comment]
Hi, Can I take this one? I have found out that `url.scheme` is `git` will return empty `tree`, I just wonder if we could add a flag on `BreadcrumbsItem` to control `breadcrumb` `clickable`,  and assign it when `FileItem` was created.

**Comment by jrieken** [comment]
No, that's not a good strategy. It may be any file system, not just `git`

**Comment by jrieken** [comment]
Actually, thinking about this more it might be an issue with underlying `git` file system. Adding @eamodio because it seems that a folder doesn't contain folders. E.g in the sample above, when clicking `bpasero` we do a read dir of its parent folder and seems to return nothing - which is wrong.

**Comment by jrieken** [comment]
Debugged into this and confirm my theory that this is a problem of the git file system provider. When resolving `/user/foo` of the `/user/foo/bar/bang.txt`-uri it states that the `/user/foo` is a file, not a folder, no children
